dev-site.yml

```diff
@@ -8,24 +8,41 @@ site:
 
 content:
   sources:
-    - url: .
-      branches: HEAD
+    - url: https://github.com/redhat-scholars/tekton-tutorial.git
+      # Stay on master to pull latest changes
+      branches: master
+      start_path: documentation
+    - url: https://github.com/kameshsampath/openshift-admins-devops.git
+      # PR branch
+      branches: tekton-pipelines
       start_path: documentation
 asciidoc:
   attributes:
-    title: OpenShift Admins DevOps(Dev Mode)
     workshop-domain: rhscholars.devx.red
+    openshift-version: 4.5
     kubernetes-version: v1.18.0
     release-version: master
+    cli: oc
     openshift-console-url: https://console-openshift-console.apps.example.com
     etherpad-url: http://example.com
+    page-pagination: true
+    tekton-version: v0.11.3
+    triggers-version: v0.4.0
+    kn-version: v0.16.0
+    tekton-cli-version: 0.11.0
+    tutorial-namespace: tektontutorial
```

Comment:
I see a lot of version strings here, but they're divorced from your actual content, because all your content is loaded via includes. It's hard to know, without building the whole site locally, what these really mean.

Reply:
Agree with that. Versioning is a good idea though.

Reply:
These are attributes used inside the content that gets substituted when `{ attribute name}` . I thought specifying the versions here will make it easier for the collaborators to quickly know what versions of software are used.